### ISSUE TITLE
Grafana SQL datasource is not working

### ISSUE THREAD

**Issue by paulfantom** [timescale/tobs#447]
**What happened?**

Grafana SQL datasource is not working and panels using this DS are empty. Most likely related to #443

**Did you expect to see something different?**

I would expect all panels to be up and running

**How to reproduce it (as minimally and precisely as possible)**:

1. Start a cluster
2. Open grafana (for example with port-forward)
3. Go to datasources setup and select SQL datasource
4. Click blue button "save and test"

**Environment**

* tobs installation method:

- [ ] cli
- [x] helm chart

* tobs version:

  `0.11.2`

    

    

* Kubernetes version information:

  `1.24`

    

* Kubernetes cluster kind:

  `kind create cluster`

**Anything else we need to know?**:

**Comment by Davincible** [comment]
By removing the scripts, you also removed the datasource SQL account setup of the Grafana reader.

```sql
    DO \$\$
      BEGIN
        CREATE ROLE prom_reader;
      EXCEPTION WHEN duplicate_object THEN
        RAISE NOTICE 'role prom_reader already exists, skipping create';
      END
    \$\$;
    DO \$\$
      BEGIN
        CREATE ROLE $GF_READER WITH LOGIN PASSWORD \'$GF_DS_PASS\';
      EXCEPTION WHEN duplicate_object THEN
        RAISE NOTICE \'role $GF_READER already exists, skipping create\';
      END
    \$\$;
    GRANT prom_reader TO $GF_DS_USER;

```

**Comment by paulfantom** [comment]
Thanks @Davincible. We are bringing it back along some tests to prevent such issues in the long run.

**Comment by paulfantom** [comment]
Closing as the fix (https://github.com/timescale/tobs/pull/450) is included in latest tobs patch release (0.11.3).
